app/javascript/stylesheets/pages/singleClass.scss

```diff
@@ -0,0 +1,209 @@
+.path-wrapper {
+  background-color: $shadow-color;
+  display: flex;
+  padding-left: 1rem;
+  line-height: 1rem;
+}
+
+.path {
+  color: $dark-color;
+  text-decoration: none;
+  cursor: pointer;
+  font-size: 0.75rem;
+  padding: 0.5rem 0;
+}
+
+.class-header {
+  background-color: var(--colorCategory);
+  color: $white-color;
+  padding: 1.25rem 0 1.25rem 1rem;
+
+  &-title {
+    font-size: 1.625rem;
+    font-weight: 600;
+    line-height: 2.25rem;
+    padding-bottom: 0.75rem;
+  }
+
+  &-subtitle {
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.375rem;
+  }
+}
+.class-menu {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+}
+
+.class-btn {
+  color: $dark-color;
+  width: 100%;
+  padding: 1rem 0;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.375rem;
+  cursor: pointer;
+  border-bottom: 0.1rem solid $shadow-color;
+  &.active {
+    border-bottom: 0.3rem solid var(--colorCategory);
+    font-weight: 600;
+  }
+}
+
+.class-about {
+  display: block;
+  &.off {
+    display: none;
+  }
+}
+
+.class-review {
+  display: block;
+  &.off {
+    display: none;
+  }
+}
+
+.class-details {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-bottom: 0.1rem solid $shadow-color;
+  color: $dark-color;
+}
+
+.class-difficulty {
+  text-transform: uppercase;
+  font-size: 0.875rem;
+  letter-spacing: 0.05rem;
+  width: 33%;
+  display: flex;
+  justify-content: center;
+}
+.class-lessons {
+  width: 33%;
+  padding: 1rem 0;
+}
+
+.class-time {
+  width: 33%;
+  padding: 1rem 0;
+}
+
+.class-details-title {
+  text-align: center;
+  font-size: 1.313rem;
```

Comment:
**question (blocking)**: why these very magic numbers? what's the base value you are using? If it's 16px then this would be 21.008px 🤔

Reply:
I'm using a convertor and only copy paste it, but the base value it's 16px and yes, this is suppose to be 21px. I'll review